packages/bundlers/experimental/src/ExperimentalBundler.js

```diff
@@ -850,9 +812,51 @@ function createIdealGraph(
       }
     }
     if (canDelete) {
+      attachOrphans(bundleRoot);
       deleteBundle(bundleRoot);
     }
   }
+
```

Comment:
Question: With moving merging of type change bundles (to accommodate for any bundle group changes in internalization), is there a scenario where we somehow inadvertently change reachability? I think no because we're only merging stuff in the same bundleGroup here.

Reply:
cc: @devongovett